### DISCUSSION TITLE
dual A770 can support p2p

### DISCUSSION THREAD

**Discussion by alanzhai219** [intel/llvm#15831] [Q&A] [answered]
### Describe the bug

as titled

### To reproduce

https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/USM/P2P/p2p_access.cpp

### Environment

ubuntu level_zero

### Additional context

BTW, cant file a Q&A

**Comment by KornevNikita** [comment]
@alanzhai219 hi! Could you please provide some more context? Are you reporting a bug or just asking something? Thank you.

**Comment by alanzhai219** [comment]
> @alanzhai219 hi! Could you please provide some more context? Are you reporting a bug or just asking something? Thank you.

ask something.

**Reply by JackAKirk** [reply]
If you have problems with P2P in the level_zero backend then your best bet is to ask @nrspruit

**Comment by KornevNikita** [comment]
Ah, okay, got it. Moved to discussions. Please re-open the issue if needed.

@JackAKirk hi, could you please provide some info here?

**Comment by jinz2014** [comment]
Can you please explain the possibility of P2P for one Intel GPU and one NVIDIA GPU ?  Thanks.

**Reply by alanzhai219** [reply]
> Can you please explain the possibility of P2P for one Intel GPU and one NVIDIA GPU ? Thanks.

- as titled: dual A770
- compiler: ipcx

**Reply by JackAKirk** [reply, marked as answer]
> Can you please explain the possibility of P2P for one Intel GPU and one NVIDIA GPU ? Thanks.

This isn't possible. P2P is only possible between two GPUs from the same backend. I've clarified this in the extension document here: https://github.com/intel/llvm/pull/15847
